### DISCUSSION TITLE
Upgrade : 1.7.8.11 to 8

### DISCUSSION THREAD

**Discussion by khasyt65** [PrestaShop/PrestaShop#36199] [Q&A]
Hi there,

I've got your 1-Click Update module (v5.0.1) but I don't find where upgrading my prestashop (1.7.8.11) to new version (8).
On the module description on the shop, it's written that it's possible to upgrade 1.7 to 8.
But 1-Click Update on my Prestashop says that I've the last version available of Prestashop ...
Can you help me please ?
Best regards,

Nathalie

**Comment by Hlavtox** [comment]
Hey, you need to select Major channel in the module, then it will offer 8.1.6 to upgrade. :-)

**Comment by khasyt65** [comment]
Hi, thank's for your answer, I'll try.
Is it possible to upgrade to 8.1.5 instead ? because my main plugin needs 8.1.5 maximum for compatibility ...

**Reply by SharakPL** [reply]
If the plugin works on 8.1.5 then it should also work on 8.1.6.
You can edit your module's main file (eg. **modules/mymodule/mymodule.php**) and change the required max version in the **__construct()** method. Here's an example on how to make it compatible with currently installed version:

![obraz](https://github.com/PrestaShop/PrestaShop/assets/5007456/4b1e1332-bdf6-4f4e-94be-18250d641aca)
